js-inputs-and-strings_switch-values/js/index.js

```diff
@@ -14,13 +14,23 @@ Hint: the HTML entity for the switch button is: &updownarrow
 */
 
 const firstInput = document.querySelector('[data-js="first-input"]');
+const secondInput = document.querySelector('[data-js="second-input"]');
 const uppercaseButton = document.querySelector('[data-js="button-uppercase"]');
 const lowercaseButton = document.querySelector('[data-js="button-lowercase"]');
+const switchButton = document.querySelector('[data-js="button-switch"]');
 
 uppercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toUpperCase();
+  secondInput.value = secondInput.value.toUpperCase();
```

Comment:
i don't think we were asked to have the uppercase button work for both input fields.
it doesn't look like a problem and the code looks good but if those weren't the instructions I wouldn't add this. otherwise looks good!